core/services/ocr2/plugins/ocr2keeper/evm21/core/utils.go

```diff
@@ -0,0 +1,26 @@
+package core
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/types"
+)
+
+// GetTxBlock calls eth_getTransactionReceipt on the eth client to obtain a tx receipt
+func GetTxBlock(client client.Client, txHash common.Hash) (*big.Int, common.Hash, error) {
+	receipt := types.Receipt{}
+	err := client.CallContext(context.Background(), &receipt, "eth_getTransactionReceipt", txHash)
+	if err != nil {
+		return nil, common.Hash{}, err
+	}
+
+	if receipt.Status != 1 {
```

Comment:
is there any documentation of what status is supposed to be in different cases. I tried to go to this struct but couldn't find much

Reply:
all i can see is [this](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_gettransactionreceipt)
it sounds like if it finds a tx, it will be 1, otherwise 0.

Reply:
thanks for the link. It seems that 0 indicates that the tx was found, but it was a failure (reverted?) 

I'm not sure if we really need special handling here for the status, as technically the tx was found on a particular block. (if it was a failure then it wouldn't have emitted a log)

but also ok to leave as is